core/src/main/java/org/apache/accumulo/core/conf/ConfigurationCopy.java

```diff
@@ -30,6 +30,8 @@
  * configuration
  */
 public class ConfigurationCopy extends AccumuloConfiguration {
+
+  AccumuloConfiguration parent = null;
```

Comment:
Would be good to do something similar to this for isSet.  Like the way it all checks the parent.  I don't think it matters for this PR, but would make the use parent consistent in case this is used by other stuff.

https://github.com/apache/accumulo/blob/2890cc2c99a1ae0fd1531533df6ee75f8b2f1cfa/core/src/main/java/org/apache/accumulo/core/conf/SiteConfiguration.java#L249

Reply:
Created https://github.com/apache/accumulo/pull/4112 to update that method.